rust/src/lib.rs

```diff
@@ -21,14 +21,20 @@
 #![allow(clippy::needless_return)]
 #![allow(clippy::let_and_return)]
 
-// Clippy lints we want to suppress due to style, or simply too noisy
-// and not a priority right now.
-#![allow(clippy::missing_safety_doc)]
-#![allow(clippy::too_many_arguments)]
+// We find this makes sense at time.
+#![allow(clippy::module_inception)]
 
-// To be fixed, but remove the noise for now.
+// The match macro is not always more clear. But its use is
+// recommended where it makes sense.
```

Comment:
Based on that, should https://redmine.openinfosecfoundation.org/issues/4607 be rejected?